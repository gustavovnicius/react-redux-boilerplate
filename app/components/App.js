import React from 'react'
import { Link } from 'react-router';

const App = ({ thunkExample }) => (
  <div>
    <button onClick={thunkExample}> Click me! </button>
  </div>
)

export default App
