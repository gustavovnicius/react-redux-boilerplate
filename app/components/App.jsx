import React from 'react';
import PropTypes from 'prop-types';

const App = ({ thunkExample }) => (
  <div>
    <button onClick={thunkExample}> Click me! </button>
  </div>
);

App.propTypes = {
  thunkExample: PropTypes.func,
}

export default App;
