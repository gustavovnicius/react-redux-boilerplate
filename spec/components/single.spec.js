import React from 'react';
import renderer from 'react-test-renderer';
import App from 'components/App';

describe('App', () => {
  it('renders', () => {
    const tree = renderer.create(React.createElement(App)).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
