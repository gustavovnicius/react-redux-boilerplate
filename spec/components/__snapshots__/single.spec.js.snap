// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders 1`] = `
<div>
  <button
    onClick={undefined}
  >
     Click me! 
  </button>
</div>
`;
